lib/cqm/converter/bonnie_measure.rb

```diff
@@ -0,0 +1,161 @@
+# CQM Converter module for HDS models.
+module CQM::Converter
+  # CQM Converter class for HDS based records.
+  module BonnieMeasure
```

Comment:
Why is this a module instead of a class? Also, update the comment above this

Reply:
There is a no infrastructure that needs to be set up before conversion. Just static methods are needed, therefore just a module.